util/process_utils.py

```diff
@@ -0,0 +1,11 @@
+from app import create_app
+from multiprocessing import Process
+
+def run_with_app_context(func, args=()):
```

Comment:
This is a bit weird, wouldn't this essentially create an entire new Flask app just to get its database access? Maybe it'd be better to pass in the database connection or something?

Tbf I'm pretty sure the celery worker does a similar thing, we can discuss.

Reply:
Yeah I believe celery does this exact same thing see [here](https://github.com/wikispeedruns/wikipedia-speedruns/blob/main/celery_worker.py). Changed this to pass in db connection info anyhow

Reply:
hmm, I actually might like the flask way of doing this better. It turns out the db connection isn't thread safe. https://stackoverflow.com/questions/47163438/is-pymysql-connection-thread-safe-is-pymysql-cursor-thread-safe

Reply:
Oh wait, now that I've looked at it, I see that you pass in the db connection info. That makes sense